cluster-autoscaler/cloudprovider/clusterapi/README.md

```diff
@@ -361,6 +361,11 @@ CAPI cloudprovider, the label format is as follows:
 
 `<gpu-type>` is arbitrary.
 
+It is important to note that if you are using the `--gpu-total` flag to limit the number
+of GPU resources in your cluster that the `<gpu-type>` value must match
```

Comment:
Can we provide a concrete known-working example that demonstrates CLI flag <--> node label affinity? That would help solidify the requirement concepts here, I think.

Reply:
absolutely